package.json

```diff
@@ -54,12 +54,12 @@
     "react-chartist": "0.14.4",
     "react-click-outside": "3.0.1",
     "react-delegate-component": "1.0.0",
-    "react-draggable": "3.3.2",
```

Comment:
Are we even using this dependency?

Reply:
I was unsure about this one, didn't see it actively used here but I saw it in Kommander and wasn't sure if it's safe to remove. Thoughts?

Reply:
In kommander-ui I can see it only in `package-lock.json` and a markdown file. So I think it is not really used and should be safe to be removed.

Reply:
Done! Thanks @mstruebing